CHANGES.rst

```diff
@@ -47,6 +47,9 @@ New features
 
   Thanks to Markus Bach.
 
+* Added calculation of vector diagram contributions to 
+  :math: `$g_\mu-2$`.
```

Comment:
@Expander, github doesn't display LaTeX from reStructuredText. Do you have any recommendation for us about using it? Should we at all? Also, should we write g-2 or (g-2)/2. I know it's irrelevant but I think that one of the reasons why we were at some point confused that we print in SLHA (g-2)/2 is that everywhere in the documentation we write g-2.

Reply:
We should use `(g-2)/2` for correctness. :) Alternatively we may use `a_\mu`.

You are correct, github does currently not render the formulas. TBH, my silent hope was that this will get fixed in the (not so far) future, as github is evolving quite a bit. And I'm still waiting and hoping. :)

Reply:
Also, I forgot. @dhjjacob, please add your name to the changelog. If Markus is there, you should be too 😉

Reply:
I wall prefer if we always a_\mu as writing (g-2)/2 may look a bit odd and people could just miss the factor 2.  Certainly never write just g-2.